lua/glow.lua

```diff
@@ -40,10 +40,14 @@ local function call_install_script()
       os="Darwin"
       arch=$(uname -m)
     fi
-    [ -z "$arch" ] || [ "$arch" == "unknown" ] && arch="x86_64"
+    if [ -z "$arch" ] || [ "$arch" == "unknown" ]; then
+      arch=$(uname -m)
+      [ "$arch" == "unknown" ] && arch="x86_64"
+    fi
     filename="glow_${version}_${os}_${arch}.tar.gz"
     url="https://github.com/charmbracelet/glow/releases/download/v${version}/${filename}"
 
+    [ -d "$GOPATH/bin" ] || mkdir -p "$GOPATH/bin"
```

Comment:
isn't this folder created by default when you install Go?

Reply:
Depends on the Linux distribution. I tested this on Void Linux, where I installed via the XBPS package manager and `$GOPATH` is not created. This caused `:GlowInstall` to fail till I created the folder. If Golang is installed directly from [here](https://golang.org/doc/install), it works fine.

Reply:
bummer, makes sense to check and create that bin path but I was thinking that it was created automatically. Anyway LGTM.